libs/fixtures/data/electionGridLayoutNewHampshireAmherst/election.json

```diff
@@ -653,7 +653,7 @@
           "column": 32,
           "row": 23,
           "contestId": "State-Representatives-Hillsborough-District-34-b1012d38",
-          "writeInIndex": 2
+          "writeInIndex": 0
```

Comment:
@adghayes I expect this may conflict with the fixture updates you made in #3190. Let's figure out how to resolve. Do you have an easy method for updating the dependent fixture data (e.g. ballot package, CVRs)?

Reply:
OK, just merged #3190 before I saw this. I didn't touch this fixture specifically. I don't have easy methods for updating dependent fixture data. 

For ballot packages, I have just been regenerating by using the app itself. 

For CVRs, I haven't gotten around to replacing I don't quite have my head wrapped around what changes you'll need to make. I think this NH CVR was originally generated by hand. And, IIRC, the CVR is already out of sync with our current software because the images aren't scaled and it doesn't do WIA properly.

Reply:
Oh, I see now that I misremembered which fixture you updated.

I guess I'll just regen the ballot packages and call it a day.